calour/io.py

```diff
@@ -316,102 +314,68 @@ def read(data_file, sample_metadata_file=None, feature_metadata_file=None,
         'openms' : an OpenMS bucket table csv (rows are feature, columns are samples)
         'openms_transpose' an OpenMS bucket table csv (columns are feature, rows are samples)
         'qiime2' : a qiime2 biom table artifact (need to have qiime2 installed)
-    encoding : str or None (optional)
-        encoder for the metadata files. None (default) to use
-        pandas default encoder, str to specify encoder name (see
-        pandas.read_table() documentation)
+    sample_metadata_kwargs, feature_metadata_kwargs : dict or None (optional)
+        keyword arguments passing to :function:`pandas.read_table` when reading sample metadata
+        or feature metadata
```

Comment:
maybe add here example for encoding. i.e. such as:
for example use 'encoding:"latin-8"'

Reply:
will add